action.yml

```diff
@@ -26,6 +26,10 @@ inputs:
       description: ' Custom regexp to extract issue number from branch name.'
       required: false
       default: ''
+  fail-pr-when-jira-issue-not-found:
```

Comment:
lets rename in to 
`fail-when-jira-issue-not-found` and default should be false

Reply:
Done